### ISSUE TITLE
Cannot suspend/hybernate/move VM if usbip is in use

### ISSUE THREAD

**Issue by maxcuttins** [xcp-ng/xcp#139]
If you try to suspend/hybernate/move a VM, you'll get the a console log in the VM guest that a device refused to freeze. The hybernation hang and VM stay stuck in migrating state forever.

I guess this issue should be fixed in the upstream distribution of the VM guest and not in the hypervisor.
However it's not good that hypervisor with xen-tools installed didn't find-out that suspension has been cancelled by OS

**Comment by stormi** [comment]
Yes, the migrating state should not stay stuck forever. Can you report it to the XenServer project? Yes, I know I write that a lot, but we get lots of interesting bug reports here that are worth of interest to them too, and we can hope that they might fix at least some of them :)

**Comment by stormi** [comment]
Hi. Have reported the issue to bugs.xenserver.org as suggested in the last comment?
